src/legacy/ui/public/new_platform/index.test.mocks.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const mockNewPlatformBackdoor = () => {
+  return {
+    __setup__: () => {},
+    __start__: () => {},
+    npSetup: {
+      core: {},
+      plugins: {
+        data: {
```

Comment:
I am planning to work on a set of "official" mocks for the data plugin; perhaps eventually these should be imported to this file from the data plugin itself, rather than having the mocks written here.

Doesn't hurt to have them here for now, just leaving a note for myself :)